benchmark/src/runner.rs

```diff
@@ -67,30 +68,22 @@ impl Stats {
     }
 
     /// Summary of stats
+    #[allow(clippy::let_underscore_untyped)]
```

Comment:
In fact, we can remove this lint.
```suggestion
    /// Summary of stats
    #[allow(clippy::let_underscore_must_use)] // the writeln! will not fail thus always return an Ok
    pub fn summary(&self) -> String {
        let mut s = String::from("\nSummary:\n");
        _ = writeln!(s, "  Total:        {:.4} secs", self.total.as_secs_f64());
        _ = writeln!(s, "  Slowest:      {:.4} secs", self.slowest.as_secs_f64());
        _ = writeln!(s, "  Fastest:      {:.4} secs", self.fastest.as_secs_f64());
        _ = writeln!(s, "  Average:      {:.4} secs", self.avg.as_secs_f64());
        _ = writeln!(s, "  Requests/sec: {:.2}", self.qps);
        s
    }
```

Reply:
done